executor/models/curvenet/curvenet.py

```diff
@@ -0,0 +1,156 @@
+"""Adapted from: https://github.com/tiangexiang/CurveNet/blob/main/core/models/curvenet_cls.py
+"""
+
+import torch.nn as nn
+import torch.nn.functional as F
+
+from .curvenet_utils import *
```

Comment:
I recommend use the specific function names instead of `*` here. It's easier to maintain.